swift/Todo/Model/DB.swift

```diff
@@ -216,7 +214,7 @@ public class DB {
     // MARK: Lists
     
     public func createTaskList(name: String) throws {
-        let docID = currentUser() + "." + NSUUID().uuidString
+        let docID = UUID().uuidString
```

Comment:
The current sync-function expects doc ID in `currentUser() + "." + NSUUID().uuidString` format.

Reply:
Same as above, I did this because `UUID` type is very strict. Changing `IResult::id` to String to fix this